src/eventbridge/send.ts

```diff
@@ -32,17 +33,16 @@ const sendEvents = async (testResults: TestActivity[]): Promise<SendResponse> =>
 
     try {
       logger.debug(`event about to be sent: ${JSON.stringify(params)}`);
-      if (testResults[i].testTypeEndTimestamp !== '') {
```

Comment:
Filtering based on the testTypeEndTimestamp wasn't done within the dynamo stream, not sure whether it needs to be kept in.
![image](https://user-images.githubusercontent.com/92087051/189629809-0e566269-fc49-42df-ada7-31529b0be2d0.png)

Reply:
From what I remember, we had this check thinking that VTA would be periodically updating the dynamo record while the testing was being carried out. What actually happens is VTA stores everything locally and will only insert into the dynamo table once the testing is complete.

Reply:
I think it may have even been that we used `testTypeEndTimestamp !== ''` as a check for whether a test had been cancelled. But then we extracted it into the dynamo filter